2.0/s2i/bin/run

```diff
@@ -10,6 +10,16 @@ set -e
 export RACK_ENV=${RACK_ENV:-"production"}
 
 if is_puma_installed; then
+  export_str=$(cgroup-limits.py)
```

Comment:
i think it is safe to do just `export $(cgroup-limits.py)` now because it will never be empty (it will print the maximum memory ;-)